geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/DescriptiveStats.scala

```diff
@@ -16,22 +16,22 @@ import scala.collection.JavaConverters._
 import scala.collection.immutable.ListMap
 import scala.Array._
 
-class DescriptiveStats(val attributes: Seq[Int]) extends Stat with Serializable {
+case class DescriptiveStats(val attributes: Seq[Int]) extends Stat with Serializable {
```

Comment:
Use of a `case` class with the `copy()` method isn't doing what you want.   Will email detailed test results.